template-config.json

```diff
@@ -3,7 +3,7 @@
     "subscription": "",
     "res_grp_name": "",
     "location": "",
-    "node_count": "",
+    "node_count": 3,
```

Comment:
I think 3 nodes goes above the Free Trial subscription limit. I'd put 1 or 2, just in case someone is testing this and finds a problem with the Free Subscription.

Reply:
Good catch!!